torchtune/config/_utils.py

```diff
@@ -12,28 +12,17 @@
 from omegaconf import DictConfig, OmegaConf
 
 from torchtune.config._errors import InstantiationError
-from torchtune.data._prompt_templates import (
-    _TemplateType,
-    PromptTemplate,
-    PromptTemplateInterface,
-)
-from torchtune.training._distributed import get_world_size_and_rank
 from torchtune.utils.logging import get_logger
 
 
 def log_config(recipe_name: str, cfg: DictConfig) -> None:
     """
-    Logs the resolved config (merged YAML file and CLI overrides) to rank zero.
+    Logs the resolved config (merged YAML file and CLI overrides).
 
     Args:
         recipe_name (str): name of the recipe to display
         cfg (DictConfig): parsed config object
     """
-    # Log the config only on rank 0
-    _, rank = get_world_size_and_rank()
```

Comment:
rank = torch.distributed.get_rank() if torch.distributed.is_available() and torch.distributed.is_initialized() else 0